torch/quantization/_adaround.py

```diff
@@ -0,0 +1,260 @@
+import torch
+import torch.nn as nn
+import torch.nn.init as init
+import math
+from torch.quantization.fake_quantize import FakeQuantize
+from torch.quantization.observer import HistogramObserver
+from torch.quantization.qconfig import *
+from torch.quantization.fake_quantize import *
+import torch.nn.qat.modules as nnqat
+from torch.quantization.default_mappings import DEFAULT_QAT_MODULE_MAPPING
+from torch.quantization import QuantStub, DeQuantStub
+import copy
+_supported_modules = {nn.Conv2d, nn.Linear}
+
+# Hyper parameters for loss function
+beta_high = 8
```

Comment:
for constant we typically use all capitable case, e.g. BETA_HIGH = 8

Reply:
also some of these parameters should probably be configurable in the adaround module I think